main.py

```diff
@@ -32,7 +32,7 @@ def get(self):
         # TODO: Generate this from a single source of truth at startup time
         context_data = {
             "dataset": {
-                "languages": ["python3", "julia"],
+                "languages": ["python3", "julia-1.9"],
```

Comment:
Definitely could/should have been more clear, but at least for now, the language name needs to match the name of the kernel, and I thought that for all versions of Julia the kernel name is just `julia`. Python has `python` and `python3` kernels because, you know, breaking changes and being able to run both kernels in one notebook.
Can you double check the kernel name for Julia?

Reply:
Ok... so this isn't a hard rule, but it's conflated in the code unless you did a change I haven't uncovered yet. We can be more specific, but we'd need a mapping somewhere, I think.